### DISCUSSION TITLE
Wifi Reconnection issues

### DISCUSSION THREAD

**Discussion by guedesopaulo** [Hieromon/AutoConnect#321] [Q&A] [answered]
When my wi-fi goes down for a moment I cannot reconnect to this wi-fi, or any other one that I have connected before, or even when that wi-fi goes back to work I cannot reconnect, even with "autoReconnect = true". What can I do to reconnect propearly?

**Comment by Hieromon** [comment, marked as answer]
Try combining the `AutoConnectConfig::reconnectInterval` setting with the autoReconnect.
A reconenctInterval setting allows AutoConnect to try autoReconnect even in the loop of a AutoConnect::handleClient.
See the [Automatic reconnect (Background)](https://hieromon.github.io/AutoConnect/adconnection.html#automatic-reconnect-background) section of the [documentation](https://hieromon.github.io/AutoConnect) for more details.

**Reply by guedesopaulo** [reply]
Sorry for the delay. What happened is while using EPROM, if any credintial was deleted, the automatic reconnect would not work. But changing to the default (nvs) things are working good now. Thank you for the library

**Reply by mellowmonk** [reply]
Im facing a similar issue, do you use esp32?

**Reply by mellowmonk** [reply]
Alright, i got it working.
  
**Config.reconnectInterval :**
Doesnt work for me, probably cos im still using the old library :-p
